cves/kernel/CVE-2018-10940.yml

```diff
@@ -448,7 +475,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    I believe the mistake here is the unintended check for a wrong arg type. The initial casting of the arg being checked is unsigned int,
+    which is being checked against an int. When actually, the arg is an unsigned long. Int -> long is double the capacity.
```

Comment:
I would also add somewhere that this is a "slip" type mistake.

Reply:
What do you mean by a "slip" type mistake? Like, a mistake that got overlooked/slipped past developers?